molecule/shared/substrate/lxd/create.yml

```diff
@@ -0,0 +1,84 @@
+---
+- name: Manage LXD containers
+  hosts: localhost
+  gather_facts: false
+  connection: local
+  vars:
+    molecule_inventory:
+      molecule:
+        hosts: {}
+    molecule_yml:
+      platforms:
+        - name: instance
+          image: ubuntu:20.04
+          user: ansible
+
+  tasks:
+    - name: Ensure LXD is installed
+      ansible.builtin.command:
+        cmd: snap install lxd
+      become: true
+
+    - name: Initialize LXD (if not already initialized)
+      command: lxd init --auto
+      args:
+        creates: /var/lib/lxd/lxd.db
+      become: true
+
```

Comment:
Should add the following to ensure Pulumi ESC gets set in the `cloud-init` file.

```yml
    - name: Get ESC from Pulumi
      ansible.builtin.command: esc open deeep-network/dev/services --format dotenv
      changed_when: false
      register: pulumi_esc
```

Reply:
added without testing